### PR TITLE
Fr11.1.2 & FR11.1.3  - help command line argument

### PR DESCRIPTION
**Pull request by kimoshinowo**
- FR11.1.2 The client should accept command line arguments “--help” and “-h” to list instructions for use and possible commands
- FR11.1.3 The server should accept command line arguments “--help” and “-h” to list instructions for use and possible commands
- Fix tabs in getHelp() method

### CODE REVIEW

**Comment by kimoshinowo** [comment]
@Jabul99 I just want to check that what I've started doing it what you meant? :)

File: newbank/server/NewBank.java [hunk 1]
```diff
@@ -108,55 +108,92 @@ public synchronized String processRequest(CustomerID customer, String request) {
...
             String command = stringInputs[0];
 
-            switch (command) {
-                case "SHOWMYACCOUNTS":
-                    return showMyAccounts(customer);
-                case "NEWACCOUNT": {
-                    if (stringInputs.length > 1) {
-                        String name = stringInputs[1];
-                        return createAccount(customer, name);
+            if (stringInputs.length == 2 && (stringInputs[1].equals("--help") || stringInputs[1].equals("-h"))) {
```
Comment (Jabul99) [inline newbank/server/NewBank.java:111]: Nicely done.

File: newbank/server/NewBank.java [hunk 2]
```diff
@@ -368,18 +405,57 @@ private String generateAccountNumber() {
...
-                "MOVE <Amount> <Debit account> <Credit account>\tMoves the amount specified between two of a customer's accounts\n" +
-                "PAY <Amount> <Debit account> <Credit account>\tPays funds from one account to another account, which may be held by another customer\n" +
-                "OFFERLOAN <Amount> <FromAccount> <Terms> <intrest>\tCreates a loan for the specified period, under the defined conditions\n" +
-                "SHOWMYOFFEREDLOANS\t\t\tShows all offered loans of the current customer\n" +
-                "SHOWOPENLOANS\t\t\tShows all open loans with the conditions of the loan.\n" +
-                "ACCEPTLOAN <Loan Number> <Account>\tThe open loan is accepted and the amount is credited to the given account.\n" +
-                "PAYBACKLOAN <Loan Number>\t\tThe loan is repaid with interest\n" +
-                "SHOWTAKENLOANS\t\tShows all taken loans of the current customer\n" +
-                "HELP\t\t\tShows this menu\n\n";
+                getHelpShowMyAccounts() +
```
Comment (Jabul99) [inline newbank/server/NewBank.java:408]: A really good choice

**Comment by Rubberduck1** [comment]
Great implementation.

**Pull request merged**